### DISCUSSION TITLE
add a specific badge

### DISCUSSION THREAD

**Discussion by b2iAlexis** [squidfunk/mkdocs-material#6465] [Q&A] [answered]
Hi,

I'm currently using Mkdocs-material for my projet documentation. I would like to add a specific
badge to my Mkdocs-material docs, but after some research, I haven't found clear information on how to do so.

![image](https://github.com/squidfunk/mkdocs-material/assets/153270775/ed703198-02d6-4548-952b-aa57a41c6ca6)

Here is the type of badge I'd like to integrate into my documentation. I would greatly appreciate it
if someone could guide me on the best way to do this, especially if this integration is
related to Mkdocs-material.

Thanks you for any assistance.

**Comment by alexvoss** [comment, marked as answer]
This [previous discussion](https://github.com/squidfunk/mkdocs-material/discussions/6286) might help. Unfortunately, these tags/badges are not a feature of Material for MkDocs (yet) but a customization built for the documentation.

**Reply by b2iAlexis** [reply]
Thank you for your reply !
This previous discussion is helpful
